.travis.yml

```diff
@@ -1,27 +0,0 @@
-language: php
```

Comment:
In `.coverall.yml`

```
service_name: travis-ci
coverage_clover: tests/tmp/clover.xml
json_path: tests/tmp/coveralls.json
```

Should we rename the service ? @pepakriz

Reply:
Coverall integration is not great currently. I'm thinking to remove it in this PR and make a better solution as a separate PR.

Reply:
Ok !